### ISSUE TITLE
games list

### ISSUE THREAD

**Issue by ReenigneArcher** [OpenXbox/xbox-webapi-python#54]
Can xbox-webapi-python do the following?

- power on/off the xbox
- provide a list of installed games and apps
  - launch those games and apps
- provide a list of owned games and apps
  - install owned games and apps that are not installed

If so are there any examples available?

If not do you know of a python module that can do these functions? I know Home Assistant has an integration (https://github.com/home-assistant/core/tree/dev/homeassistant/components/xbox) that can do the first three items in my list, but I'd rather not have to reverse engineer their integration and have to strip out all of their entity and sensor stuff.

**Comment by uxm5jc** [comment]
I'm not 100% on the checking your list of games but you can definitely search up games and power on/off the xbox. _its also in like the first paragraph of the readme file..._

**Comment by ReenigneArcher** [comment]
> I'm not 100% on the checking your list of games but you can definitely search up games and power on/off the xbox. _its also in like the first paragraph of the readme file..._

I apologize, but I didn't see answers to my questions in the readme. This is the first paragraph which doesn't specifically mention any of the items I asked about.

```
Xbox-WebAPI is a python library to authenticate with Xbox Live via your Microsoft Account and provides Xbox related Web-API.
```

Since I created this issue I did find the smartglass core repository here and was able to power on/off the console and get info about the recently (all) played games. I didn't see a way to get a list of currently installed games/apps or owned games/apps. Launching also didn't appear to work. I'll keep digging and see if I can figure out how home assistant is doing these things.

Thanks.
